### DISCUSSION TITLE
尊敬的开发者,是否可以添加lubancat2(鲁班猫2)系列的支持呢?

### DISCUSSION THREAD

**Discussion by Ethanwain** [ophub/amlogic-s9xxx-armbian#2047] [Rockchip]
需要我提供什么资料吗

**Comment by Ethanwain** [comment]
[rk3568-lubancat-2.dtb.zip](https://github.com/ophub/amlogic-s9xxx-armbian/files/14053494/rk3568-lubancat-2.dtb.zip)
我有lubancat2的dtb文件,我也尝试替换dtb来启动armbian,请问这样替换系统是否稳定?

**Reply by Lozy** [reply]
Hi. 请问你这个 dtb是从哪里提取的，是否有 2N的 dtb

**Comment by ophub** [comment]
dtb符合设备，系统就稳定，你可以在TF卡上替换测试。可以挂机观察可以压力测试。稳定不稳定不是谁能一句话给设备定调的，硬件、系统、软件、使用共同作用。

**Reply by Ethanwain** [reply]
嗯 目前测试还OK,我过两天测试下应用跑起来如何,感谢解答

**Reply by Ethanwain** [reply]
我测试了替换dtb的系统,发现开发板串口没法开通,厂家的debian10系统是通过dtbo开启对应的各个uart串口,请问如何添加dtbo开启串口呀?
我上传了厂家debian10系统的boot目录,对了,这个系统的dtb替换没法启动armbian
[boot.zip](https://github.com/ophub/amlogic-s9xxx-armbian/files/14156431/boot.zip)

**Comment by fuyb** [comment]
请问是用哪个镜像做的测试呢？

**Reply by Ethanwain** [reply]
r5c测试的,但是串口号对不上....这个系统的串口号是ttyS1-S5 而鲁班猫2的串口号是3.4.7.9,我也不知道怎么修改
